integrations/c/install-driver.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+echo "Running C Driver install-driver.sh..."
+
+# Defined by .evergreen/config.yml.
+echo "CC: ${CC:?}"
+echo "CXX: ${CXX:?}"
+
+# Ensure required binaries are present.
+command -V "${CC}" >/dev/null || exit
+command -V "${CXX}" >/dev/null || exit
+
```

Comment:
Given the C Driver integration is (deliberately) specifically on ubuntu1804-drivers-atlas-testing only, I omitted checks and tests for distro-compatibility. If we ever get around to supporting more distros, then a check similar to this would indeed be necessary.